Objects/rangeobject.c

```diff
@@ -966,6 +982,9 @@ fast_range_iter(long start, long stop, long step, long len)
     it->start = start;
     it->step = step;
     it->len = len;
+#ifdef Py_GIL_DISABLED
+    it->stop = stop;
+#endif
```

Comment:
I'm not enthusiastic about the idea of growing the iterator size. Can we unconditionally replace `len` with `stop`?